### ISSUE TITLE
flag autopowerspec flagging fails

### ISSUE THREAD

**Issue by mpatir** [caracal-pipeline/caracal#1193]
flag_autopowerspec: fails and complains about that it could not find required "field", but the command seems to have the correct fields.  I am using it in the default configuration:

Config file:

  flag_autopowerspec:
    enable: true

Does anybody know if there a way to fix this? 

Traceback (most recent call last):
File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
     "__main__", fname, loader, pkg_name)
   File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
     exec code in run_globals
   File "/usr/local/lib/python2.7/dist-packages/politsiyakat/__main__.py", line 25, in <module>
     politsiyakat.main(sys.argv[1:])
   File "/usr/local/lib/python2.7/dist-packages/politsiyakat/__init__.py", line 137, in main
     run_func(**args.kwargs)
   File "/usr/local/lib/python2.7/dist-packages/politsiyakat/modules/flag_tasks.py", line 589, in flag_autocorr_drifts
     raise ValueError("flag_autocorr_drifts expects field(s) (key "
 ValueError: flag_autocorr_drifts expects field(s) (key 'field') as input
 Traceback (most recent call last):
   File "/stimela_mount/code/run.py", line 39, in <module>
     subprocess.check_call(shlex.split(_runc))
   File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
     raise CalledProcessError(retcode, cmd)
 subprocess.CalledProcessError: Command '['python', '-m', 'politsiyakat', 'flag_autocorr_drifts', '-s', 'antenna_mod', '{"plot_size": 6, "nrows_chunk": 5000, "data_column": "DATA", "scan_to_scan_threshold": 3, "cal_field": "J1257-3155,3C286", "nio_threads": 1, "field": "J1257-3155,3C286", "output_dir": "./", "simulate": false, "msname": "/stimela_mount/msdir/J135_1_cal.ms", "nproc_threads": 8, "dpi": 300, "antenna_to_group_threshold": 5}']' returned non-zero exit status

**Comment by KshitijT** [comment]
It probably needs field-id, not field-name. Politsiyakat is, I think, available on pypi - could you try a run outside CARACal?

**Comment by mpatir** [comment]
@KshitijT 
Ah okay I will try it outside the pipeline. There is no way to specify field-id in caracal.

**Comment by KshitijT** [comment]
> There is no way to specify field-id in caracal.

Just to confirm it's a bug.

**Comment by mpatir** [comment]
not winning with installing politsiyakat. It just is not installing with "pip install politsiyakat". I will open an issue on its Github page. 

In the meantime, I found that this was an issue, which was fixed 10 months ago. 
https://github.com/bennahugo/politsiyakat/issues/7

I will check if running it outside the pipeline works or not, once I figure out how to install it

**Comment by KshitijT** [comment]
> not winning with installing politsiyakat. It just is not installing with "pip install politsiyakat". I will open an issue on its Github page.

Please use a python 2.7 virtualenv, it installs fine.

>In the meantime, I found that this was an issue, which was fixed 10 months ago.
bennahugo/politsiyakat#7

I don't think that's the same issue; in @SpheMakh 's comment Politsiyakat has already selected the field, right?

**Comment by mpatir** [comment]
> > not winning with installing politsiyakat. It just is not installing with "pip install politsiyakat". I will open an issue on its Github page.
> 
> Please use a python 2.7 virtualenv, it installs fine.
ramatsoku@machine:~/msdir$ virtualenv politsiyakatvenv

Running virtualenv with interpreter /usr/bin/python2
New python executable in /home/ramatsoku/msdir/politsiyakatvenv/bin/python2
Also creating executable in /home/ramatsoku/msdir/politsiyakatvenv/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.

ramatsoku@machine:~/msdir$ source politsiyakatvenv/bin/activate

(politsiyakatvenv) ramatsoku@machine:~/msdir$ pip install politsiyakat

Collecting politsiyakat
  Using cached politsiyakat-0.3.9-py2-none-any.whl (42 kB)
Collecting python-casacore>=2.1.2
  Using cached python_casacore-3.3.1-cp27-cp27mu-manylinux2010_x86_64.whl (25.5 MB)
Collecting matplotlib>=1.5.0
  Using cached matplotlib-2.2.5-cp27-cp27mu-manylinux1_x86_64.whl (12.8 MB)
Collecting scipy>=1.0.0
  Using cached scipy-1.2.3-cp27-cp27mu-manylinux1_x86_64.whl (24.8 MB)
Collecting futures
  Using cached futures-3.3.0-py2-none-any.whl (16 kB)
Collecting numpy>=1.13.3
  Using cached numpy-1.16.6-cp27-cp27mu-manylinux1_x86_64.whl (17.0 MB)
Collecting progress
  Using cached progress-1.5.tar.gz (5.8 kB)
Collecting six
  Using cached six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting future
  Using cached future-0.18.2.tar.gz (829 kB)
Requirement already satisfied: argparse in /usr/lib/python2.7 (from python-casacore>=2.1.2->politsiyakat) (1.2.1)
Collecting python-dateutil>=2.1
  Using cached python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting backports.functools-lru-cache
  Using cached backports.functools_lru_cache-1.6.1-py2.py3-none-any.whl (5.7 kB)
Collecting pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1
  Using cached pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
Collecting cycler>=0.10
  Using cached cycler-0.10.0-py2.py3-none-any.whl (6.5 kB)
Collecting kiwisolver>=1.0.1
  Using cached kiwisolver-1.1.0-cp27-cp27mu-manylinux1_x86_64.whl (93 kB)
Collecting pytz
  Using cached pytz-2020.1-py2.py3-none-any.whl (510 kB)
Collecting subprocess32
  Using cached subprocess32-3.5.4-cp27-cp27mu-manylinux2014_x86_64.whl (69 kB)
Requirement already satisfied: setuptools in ./politsiyakatvenv/lib/python2.7/site-packages (from kiwisolver>=1.0.1->matplotlib>=1.5.0->politsiyakat) (45.0.0)
ERROR: Package 'setuptools' requires a different Python: 2.7.12 not in '>=3.5'

(politsiyakatvenv) ramatsoku@machine:~/msdir$ politsiyakat
politsiyakat: command not found

(politsiyakatvenv) ramatsoku@machine:~/msdir$ python -m politsiyakat

/home/ramatsoku/msdir/politsiyakatvenv/bin/python: No module named politsiyakat

(politsiyakatvenv) ramatsoku@machine:~/msdir$ 

> 
> > In the meantime, I found that this was an issue, which was fixed 10 months ago.
> > [bennahugo/politsiyakat#7](https://github.com/bennahugo/politsiyakat/issues/7)
> 
> I don't think that's the same issue; in @SpheMakh 's comment Politsiyakat has already selected the field, right?

Ah yes, you are right, in that issue the field was already selected.

**Comment by KshitijT** [comment]
Damn. 

The same thing went like this for me (half an hour ago):

```
kshitij@stills:~$ virtualenv test-venv
Running virtualenv with interpreter /usr/bin/python2
New python executable in /home/kshitij/test-venv/bin/python2
Also creating executable in /home/kshitij/test-venv/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.
kshitij@stills:~$ source test-venv/
-bash: source: test-venv/: is a directory
kshitij@stills:~$ source test-venv/bin/activate
(test-venv) kshitij@stills:~$ pip install politsiyakat
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Collecting politsiyakat
  Downloading politsiyakat-0.3.9-py2-none-any.whl (42 kB)
     |████████████████████████████████| 42 kB 658 kB/s 
Collecting python-casacore>=2.1.2
  Downloading python_casacore-3.3.1-cp27-cp27mu-manylinux2010_x86_64.whl (25.5 MB)
     |████████████████████████████████| 25.5 MB 23.5 MB/s 
Collecting scipy>=1.0.0
  Downloading scipy-1.2.3-cp27-cp27mu-manylinux1_x86_64.whl (24.8 MB)
     |████████████████████████████████| 24.8 MB 40.2 MB/s 
Collecting futures
  Using cached futures-3.3.0-py2-none-any.whl (16 kB)
Collecting matplotlib>=1.5.0
  Downloading matplotlib-2.2.5-cp27-cp27mu-manylinux1_x86_64.whl (12.8 MB)
     |████████████████████████████████| 12.8 MB 62.7 MB/s 
Collecting numpy>=1.13.3
  Using cached numpy-1.16.6-cp27-cp27mu-manylinux1_x86_64.whl (17.0 MB)
Collecting progress
  Downloading progress-1.5.tar.gz (5.8 kB)
Collecting six
  Using cached six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting future
  Using cached future-0.18.2.tar.gz (829 kB)
Requirement already satisfied: argparse in /usr/lib/python2.7 (from python-casacore>=2.1.2->politsiyakat) (1.2.1)
Collecting python-dateutil>=2.1
  Using cached python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting kiwisolver>=1.0.1
  Using cached kiwisolver-1.1.0-cp27-cp27mu-manylinux1_x86_64.whl (93 kB)
Collecting cycler>=0.10
  Using cached cycler-0.10.0-py2.py3-none-any.whl (6.5 kB)
Collecting pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1
  Using cached pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
Collecting pytz
  Using cached pytz-2020.1-py2.py3-none-any.whl (510 kB)
Collecting backports.functools-lru-cache
  Using cached backports.functools_lru_cache-1.6.1-py2.py3-none-any.whl (5.7 kB)
Collecting subprocess32
  Downloading subprocess32-3.5.4-cp27-cp27mu-manylinux2014_x86_64.whl (69 kB)
     |████████████████████████████████| 69 kB 11.3 MB/s 
Requirement already satisfied: setuptools in ./test-venv/lib/python2.7/site-packages (from kiwisolver>=1.0.1->matplotlib>=1.5.0->politsiyakat) (44.1.1)
Building wheels for collected packages: progress, future
  Building wheel for progress (setup.py) ... done
  Created wheel for progress: filename=progress-1.5-py2-none-any.whl size=8072 sha256=eb873b721c9a076beeb984a2ef38a7de893b8cf9c1c66e1f462f2062bd21c4c7
  Stored in directory: /home/kshitij/.cache/pip/wheels/07/36/17/1754e511ae6586f0e70936243e3e5ef0a4fc6bae82baa55a71
  Building wheel for future (setup.py) ... done
  Created wheel for future: filename=future-0.18.2-py2-none-any.whl size=502566 sha256=40eb4683cc235b4723899c3d72ec37c01d0e61fc00a86231b67b1d6a0fd8f2fd
  Stored in directory: /home/kshitij/.cache/pip/wheels/5f/11/0c/aad680baf5ef4fbcbab992c9f03e1130357e0c173a4fdabfff
Successfully built progress future
Installing collected packages: six, numpy, future, python-casacore, scipy, futures, python-dateutil, kiwisolver, cycler, pyparsing, pytz, backports.functools-lru-cache, subprocess32, matplotlib, progress, politsiyakat
Successfully installed backports.functools-lru-cache-1.6.1 cycler-0.10.0 future-0.18.2 futures-3.3.0 kiwisolver-1.1.0 matplotlib-2.2.5 numpy-1.16.6 politsiyakat-0.3.9 progress-1.5 pyparsing-2.4.7 python-casacore-3.3.1 python-dateutil-2.8.1 pytz-2020.1 scipy-1.2.3 six-1.15.0 subprocess32-3.5.4
```

**Comment by KshitijT** [comment]
Ok, I tested this out:

This: 
```
python -m politsiyakat -s 'antenna_mod' flag_phase_drifts  '{"msname":"1477074305.subset.ms", "plot_size": 6, "nrows_chunk": 5000, "data_column": "DATA", "scan_to_scan_threshold": 3, "field": "0", "cal_field": "0", "dpi": 300, "antenna_to_group_threshold": 5, "simulate": false, "output_dir": "./", "baseline_to_group_threshold": 3}'


```
works fine: 

```
politsiyakat - 2020-07-03 16:50:32,359 INFO - Module installed at '/home/kshitij/test-venv/local/lib/python2.7/site-packages/politsiyakat' version 0.3.9
politsiyakat - 2020-07-03 16:50:32,361 WARNING - Setting maximum number of IO threads to 1. This behaviour can be controlled via parameter 'nio_threads'
politsiyakat - 2020-07-03 16:50:32,361 WARNING - Setting maximum number of worker threads to 1. This behaviour can be controlled via parameter 'nproc_threads'
politsiyakat - 2020-07-03 16:50:32,361 INFO - Running task 'flag_phase_drifts' with the following arguments:
politsiyakat - 2020-07-03 16:50:32,361 INFO - 	plot_size:6
politsiyakat - 2020-07-03 16:50:32,361 INFO - 	nrows_chunk:5000
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	data_column:DATA
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	field:0
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	cal_field:0
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	baseline_to_group_threshold:3
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	scan_to_scan_threshold:3
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	antenna_to_group_threshold:5
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	simulate:False
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	msname:1477074305.subset.ms
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	dpi:300
politsiyakat - 2020-07-03 16:50:32,362 INFO - 	output_dir:./
politsiyakat - 2020-07-03 16:50:32,376 INFO - 1477074305.subset.ms appears to be a valid measurement set with 10505 rows
politsiyakat - 2020-07-03 16:50:32,376 WARNING - Warning: defaulting to plot dpi of 300. Keyword 'plot_dpi' can be used to control this behaviour.
politsiyakat - 2020-07-03 16:50:32,376 INFO - Will process the following fields:
politsiyakat - 2020-07-03 16:50:32,376 INFO - 	(0): PKS1934-638 (calibrator)
politsiyakat - 2020-07-03 16:50:32,386 INFO - 1477074305.subset.ms appears to be a valid measurement set with 10505 rows
politsiyakat - 2020-07-03 16:50:32,430 INFO - Processing targets and calibrators in chunk 1 of 3...
politsiyakat - 2020-07-03 16:50:32,432 INFO - 	Reading MS
politsiyakat - 2020-07-03 16:50:32,464 INFO - 	Processing field:
Processing epic 'antenna stats for PKS1934-638' ◉◉◉◉◉◉◉◉◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯◯ 26%^Cpolitsiyakat - 2020-07-03 16:50:33,393 INFO - Waiting for remaining jobs to finish...

```

whereas when I replace the field-id with field-name:
```
python -m politsiyakat -s 'antenna_mod' flag_phase_drifts  '{"msname":"1477074305.subset.ms", "plot_size": 6, "nrows_chunk": 5000, "data_column": "DATA", "scan_to_scan_threshold": 3, "field": "PKS1934-638", "cal_field": "PKS1934-638", "dpi": 300, "antenna_to_group_threshold": 5, "simulate": false, "output_dir": "./", "baseline_to_group_threshold": 3}'

```
it crashes with the same error @mpatir gets:

```
politsiyakat - 2020-07-03 16:52:12,269 INFO - Module installed at '/home/kshitij/test-venv/local/lib/python2.7/site-packages/politsiyakat' version 0.3.9
politsiyakat - 2020-07-03 16:52:12,271 WARNING - Setting maximum number of IO threads to 1. This behaviour can be controlled via parameter 'nio_threads'
politsiyakat - 2020-07-03 16:52:12,271 WARNING - Setting maximum number of worker threads to 1. This behaviour can be controlled via parameter 'nproc_threads'
politsiyakat - 2020-07-03 16:52:12,271 INFO - Running task 'flag_phase_drifts' with the following arguments:
politsiyakat - 2020-07-03 16:52:12,271 INFO - 	plot_size:6
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	nrows_chunk:5000
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	data_column:DATA
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	field:PKS1934-638
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	cal_field:PKS1934-638
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	baseline_to_group_threshold:3
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	scan_to_scan_threshold:3
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	antenna_to_group_threshold:5
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	simulate:False
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	msname:1477074305.subset.ms
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	dpi:300
politsiyakat - 2020-07-03 16:52:12,272 INFO - 	output_dir:./
politsiyakat - 2020-07-03 16:52:12,285 INFO - 1477074305.subset.ms appears to be a valid measurement set with 10505 rows
politsiyakat - 2020-07-03 16:52:12,285 INFO - Waiting for remaining jobs to finish...
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/kshitij/test-venv/lib/python2.7/site-packages/politsiyakat/__main__.py", line 25, in <module>
    politsiyakat.main(sys.argv[1:])
  File "/home/kshitij/test-venv/local/lib/python2.7/site-packages/politsiyakat/__init__.py", line 137, in main
    run_func(**args.kwargs)
  File "/home/kshitij/test-venv/local/lib/python2.7/site-packages/politsiyakat/modules/flag_tasks.py", line 76, in flag_phase_drifts
    raise ValueError("flag_phase_drifts expects field(s) (key "
ValueError: flag_phase_drifts expects field(s) (key 'field') as input


```

This is a bug - we need to replace the field-name with field-id for the `field` argument.

**Comment by mpatir** [comment]
Maybe my computer has a serious case of the "Fridays" -- I have no idea why the same installation procedure fails. 

> politsiyakat - 2020-07-03 16:52:12,271 INFO - 	plot_size:6
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	nrows_chunk:5000
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	data_column:DATA
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	field:PKS1934-638
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	cal_field:PKS1934-638
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	baseline_to_group_threshold:3
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	scan_to_scan_threshold:3
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	antenna_to_group_threshold:5
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	simulate:False
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	msname:1477074305.subset.ms
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	dpi:300
> politsiyakat - 2020-07-03 16:52:12,272 INFO - 	output_dir:./
> politsiyakat - 2020-07-03 16:52:12,285 INFO - 1477074305.subset.ms appears to be a valid measurement set with 10505 rows
> politsiyakat - 2020-07-03 16:52:12,285 INFO - Waiting for remaining jobs to finish...
> Traceback (most recent call last):
>   File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
>     "__main__", fname, loader, pkg_name)
>   File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
>     exec code in run_globals
>   File "/home/kshitij/test-venv/lib/python2.7/site-packages/politsiyakat/__main__.py", line 25, in <module>
>     politsiyakat.main(sys.argv[1:])
>   File "/home/kshitij/test-venv/local/lib/python2.7/site-packages/politsiyakat/__init__.py", line 137, in main
>     run_func(**args.kwargs)
>   File "/home/kshitij/test-venv/local/lib/python2.7/site-packages/politsiyakat/modules/flag_tasks.py", line 76, in flag_phase_drifts
>     raise ValueError("flag_phase_drifts expects field(s) (key "
> ValueError: flag_phase_drifts expects field(s) (key 'field') as input
> ```
> 
> This is a bug - we need to replace the field-name with field-id for the `field` argument.

Thank you very much for testing this, ah okay so it is a bug, 
good I thought I had done something weirdly wrong

**Comment by KshitijT** [comment]
To add further to the mystery, I ran the pipeline with the defaults on with onlu flag_autopowerspec on; runs fine:

```
# politsiyakat - 2020-07-04 20:05:07,111 INFO - Module installed at '/usr/local/lib/python2.7/dist-packages/politsiyakat' version 0.3.9
# politsiyakat - 2020-07-04 20:05:07,112 INFO - Running task 'flag_autocorr_drifts' with the following arguments:
# politsiyakat - 2020-07-04 20:05:07,112 INFO - 	plot_size:6
# politsiyakat - 2020-07-04 20:05:07,112 INFO - 	nrows_chunk:5000
# politsiyakat - 2020-07-04 20:05:07,112 INFO - 	data_column:DATA
# politsiyakat - 2020-07-04 20:05:07,112 INFO - 	field:1,2,0
# politsiyakat - 2020-07-04 20:05:07,113 INFO - 	cal_field:1,2,0
# politsiyakat - 2020-07-04 20:05:07,113 INFO - 	scan_to_scan_threshold:3
# politsiyakat - 2020-07-04 20:05:07,113 INFO - 	output_dir:./
# politsiyakat - 2020-07-04 20:05:07,113 INFO - 	simulate:False
# politsiyakat - 2020-07-04 20:05:07,113 INFO - 	msname:/stimela_mount/msdir/1524947605-cal.ms
# politsiyakat - 2020-07-04 20:05:07,113 INFO - 	dpi:300
# politsiyakat - 2020-07-04 20:05:07,113 INFO - 	antenna_to_group_threshold:5
# politsiyakat - 2020-07-04 20:05:07,126 INFO - /stimela_mount/msdir/1524947605-cal.ms appears to be a valid measurement set with 39680 rows
# politsiyakat - 2020-07-04 20:05:07,127 WARNING - Warning: defaulting to plot dpi of 300. Keyword 'plot_dpi' can be used to control this behaviour.
# politsiyakat - 2020-07-04 20:05:07,127 INFO - Will process the following fields:
# politsiyakat - 2020-07-04 20:05:07,127 INFO - 	(1): J1318-4620 (calibrator)
# politsiyakat - 2020-07-04 20:05:07,127 INFO - 	(2): J1424-4913 (calibrator)
# politsiyakat - 2020-07-04 20:05:07,127 INFO - 	(0): J1331+3030 (calibrator)
# politsiyakat - 2020-07-04 20:05:07,136 INFO - /stimela_mount/msdir/1524947605-cal.ms appears to be a valid measurement set with 39680 rows
# politsiyakat - 2020-07-04 20:05:07,150 INFO - Processing chunk 1 of 8...
# politsiyakat - 2020-07-04 20:05:07,152 INFO - 	Reading MS

```
The flagging section looks like: 

```
flag:
  enable: true
  field: calibrators
  label_in: cal
  flag_autopowerspec:
    enable: true
  flag_rfi:
    enable: true
    flagger: aoflagger
  flag_autocorr:
    enable: true
  flag_shadow:
    enable: true
  report: true

```

So I am thoroughly confused now. 

Can someone test this out quickly to make sure this is not a case of Fridays? :)

**Comment by Athanaseus** [comment]
This was resolved by #1185. @mpatir just try updating to the latest master.
